.github/workflows/examples-efr32.yaml

```
@@ -38,7 +38,7 @@ jobs:
     if: github.actor != 'restyled-io[bot]'
 
     container:
-      image: ghcr.io/project-chip/chip-build-efr32:8
+      image: ghcr.io/project-chip/chip-build-efr32:18
```

```suggestion
      image: ghcr.io/project-chip/chip-build-efr32:19
```
had to create another image to fix the last one